fisrt commit

pavel-gutsal opened the pull request:
https://pavel-gutsal.github.io/react_decompose/

valerii-derkach commented on src/components/Header/Header.jsx:
```diff
@@ -1,6 +1,15 @@
+const Header = () => (
+  <header className="header">
+    <h1 className="header__title">Site Name</h1>
+    <nav className="navigation">
+      <a className="navigation__link" href="#about">About</a>
+      <a className="navigation__link" href="#services">Services</a>
+      <a className="navigation__link" href="#contact">Contact</a>
```
navigation is separate component, so create a file and move it there
render navigation inside header

valerii-derkach commented on src/components/Header/Header.css:
```diff
@@ -1 +1,42 @@
+  font-size: inherit;
+}
+
+.navigation__link {
+  display: inline-block;
+  outline: none;
+  text-decoration: none;
+  opacity: 0.7;
+  padding: 0 0.5em;
+  color: black;
+  transition: opacity 0.2s ease-in-out;
+}
+
+.navigation__link:hover,
+.navigation__link:focus {
+  opacity: 1;
+}
```
move navigation styles to separate component

pavel-gutsal commented:
Split the `App.js` into smaller components basing on CSS blocks. CSS should be split too.
- Create a sub folder in `components` per component and put `js` and `css` files there.
- There should be 3 new components: `Article`, `Header`, `Welcome`.

So I added navigation as separate component even if its not in the task , on request of the mentor , please don`t ask me to remove it)))

valerii-derkach reviewed (approved):
Everything good! Well done!
